packages/git/src/node/git-locator/git-locator-client.ts

```diff
@@ -36,7 +36,7 @@ export class GitLocatorClient implements GitLocator {
         return new Promise((resolve, reject) => {
             const toStop = this.ipcConnectionProvider.listen({
                 serverName: 'git-locator',
-                entryPoint: paths.resolve(__dirname, 'git-locator-host')
+                entryPoint: paths.join(__dirname, 'git-locator-host'),
```

Comment:
Nitpick: redundant `,` at EOL. Afaik we don't enforce a member delimiter style, so feel free to ignore this.